README.md

```diff
@@ -16,7 +16,8 @@ the square brackets (for multiple-choice questions).
 What are constructor functions and the `new` keyword? What is a prototype? Describe an example of when we would use a constructor function versus a prototype.
 
 Your Answer:
-```text
+```constructor function are functions that define properties. prototype is the properties to inhertied to the new object created by new keyword.
```

Comment:
Constructor functions don't necessarily define properties. They're the code that's run every time you create a new instance of an object.